src/Backend/SgateShopgatePlugin/Components/Export.php

```diff
@@ -218,6 +231,72 @@ public function checkCompleteCategory($categoryId)
         return in_array($categoryId, $this->languageCompleteCategoryList);
     }
 
+    /**
+     * @return phpFastCache\Core\DriverAbstract
+     */
+    protected function getCacheInstance()
+    {
+        if (!$this->cacheInstance) {
+            $this->cacheInstance = CacheManager::getInstance(
+                'files', array(
+                'path' => rtrim(
+                        Shopware()->DocPath(), DS) . DS . 'web' . DS . 'cache' . DS
```

Comment:
In 4.3.6 it's just /cache instead of /web/cache. We should not copy logic here and just use getCacheFolderPath() from Shopware_Plugins_Backend_SgateShopgatePlugin_Components_Config as we already took care of different versions, dir creation and .htaccess